expanding on windows install docs (#10788)
opensearch-project/documentation-website: _install-and-configure/install-dashboards/windows.md

      ```
   1. Restart your computer.

1. Configure OpenSearch Dashboards.

>     There are two ways to configure OpenSearch Dashboard and this depends on whether the OpenSearch is configured with security enabled or disabled.

Reviewer: ```suggestion
    There are two ways to configure OpenSearch Dashboards, depending on whether OpenSearch is configured with security enabled or disabled.
```

>     In order for any changes to `opensearch_dashboards.yml` file to take effect, a restart of OpenSearch Dashboards is required.

Reviewer: ```suggestion
    In order for any changes to the `opensearch_dashboards.yml` file to take effect, a restart of OpenSearch Dashboards is required.
```

    {: .note}

>     1. Option 1: With security enabled:

Reviewer: ```suggestion
    1. Option 1 -- With security enabled:
```

  
        Configuration file `\path\to\opensearch-dashboards-{{site.opensearch_version}}\config\opensearch_dashboards.yml` comes packaged with following basic settings:
        
        ```
        opensearch.hosts: [https://localhost:9200]
        opensearch.ssl.verificationMode: none
        opensearch.username: kibanaserver
        opensearch.password: kibanaserver
        opensearch.requestHeadersWhitelist: [authorization, securitytenant]
        
        opensearch_security.multitenancy.enabled: true
        opensearch_security.multitenancy.tenants.preferred: [Private, Global]
        opensearch_security.readonly_mode.roles: [kibana_read_only]
        # Use this setting if you are running opensearch-dashboards without https
        opensearch_security.cookie.secure: false
        ```
    

>     1. Option 2: With OpenSearch security disabled:

Reviewer: ```suggestion
    1. Option 2 -- With OpenSearch security disabled:
```

>         If you are using OpenSearch with security disabled, remove the security plugin from OpenSearch Dashboards using the following command:

Reviewer: ```suggestion
        If you are using OpenSearch with security disabled, remove the Security plugin from OpenSearch Dashboards using the following command:
```

        
        ```
        \path\to\opensearch-dashboards-{{site.opensearch_version}}\bin\opensearch-dashboards-plugin.bat remove securityDashboards
        ```
        

>         Basic `opensearch_dashboards.yml` file should contain:

Reviewer: ```suggestion
        The basic `opensearch_dashboards.yml` file should contain:
```
